TestModels/SimpleTypes/SimpleBoolean/runtimes/rust/src/operation/get_boolean/builders.rs

```diff
@@ -56,17 +59,17 @@ impl GetBooleanFluentBuilder {
     }
 
     #[allow(missing_docs)] // documentation missing in model
-    pub fn value(mut self, input: impl ::std::convert::Into<::std::primitive::bool>) -> Self {
-        self.inner = self.inner.value(input.into());
-        self
-    }
-    #[allow(missing_docs)] // documentation missing in model
-    pub fn set_value(mut self, input: ::std::option::Option<::std::primitive::bool>) -> Self {
-        self.inner = self.inner.set_value(input);
-        self
-    }
-    #[allow(missing_docs)] // documentation missing in model
-    pub fn get_value(&self) -> &::std::option::Option<::std::primitive::bool> {
-        self.inner.get_value()
-    }
+pub fn value(mut self, input: impl ::std::convert::Into<::std::primitive::bool>) -> Self {
```

Comment:
We should run `rustfmt` after generating, as we do for other languages (but feel free to fast-follow on that)

Reply:
I'll fast-follow on adding this.